settings/language-sqf.json

```diff
@@ -1,5 +1,5 @@
 {
-  ".source.sqf, .source.cpp, .source.hpp, .source.ext": {
+  ".source.sqf, .source.arma.cfg": {
```

Comment:
I am not sure I like this, you need to have "SQF" selected for that to work, by default those files have "C++" selected and it's just an unnecessary step to choose it. I'd leave it as it was.